compiler/compile/OMRCompilation.hpp

```diff
@@ -676,6 +676,36 @@ class OMR_EXTENSIBLE Compilation
 
    void setIlVerifier(TR::IlVerifier *ilVerifier) { _ilVerifier = ilVerifier; }
 
+   typedef std::pair<void *, TR::DebugCounterBase *> DebugCounterEntry;
+   typedef TR::typed_allocator<DebugCounterEntry, TR::Allocator> DebugCounterMapAllocator;
+   typedef std::map<void *, TR::DebugCounterBase *, std::less<void *>, DebugCounterMapAllocator> DebugCounterMap;
```

Comment:
please make the key type "const" to avoid compile failures on more recent compilers